<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogSettingsCustom</class>
 <widget class="QDialog" name="dialogSettingsCustom">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Custom Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#dialogSettingsCustom{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.4 rgba(60, 153, 170, 255), stop:1 rgba(17, 99, 132, 255));
}
</string>
  </property>
  <widget class="QLabel" name="labelCustomDepartment">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="text">
    <string>Department</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCustomStorage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="text">
    <string>Storage</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCustomCategory">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCustomEmployee">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="text">
    <string>Employee</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCustomDepartment">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Name of department</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCustomStorage">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Name of storage</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCustomCategory">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Name of category</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCustomEmployee">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Name of employee</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCustomDepartmentAdd">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCustomStorageAdd">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>120</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCustomCategoryAdd">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCustomEmployeeAdd">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>280</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
